.github/workflows/publish.yml

```diff
@@ -0,0 +1,78 @@
+---
+name: Publish on Test PyPI and PyPI
+
+on:
+    push:
+        branches:
+      # Commits pushed to release/ branches are published on Test PyPI if they
+      # have a new version number.
+            - release/**
+        tags:
+      # Tags that start with the "v" prefix are published on PyPI.
+            - v*
+
+jobs:
+
+    build:
+
+        runs-on: ubuntu-latest
+
+        steps:
+
+            - uses: actions/checkout@v2
+
+            - name: Set up Python 3.10
+              uses: actions/setup-python@v2
+              with:
+                  python-version: '3.10'
+
+            - name: Install pypa/build
```

Comment:
I think installation of build package is not enough. The package needs npm, jupyter_packaging, yarn etc to build the Javascript library.

Reply:
You mean without JS packages the next step would fail? Can you add me as the owner of PyPI so I can test with a true release let's say `0.4.1a0`, and if it failed, I can delete it?

Reply:
Yes, you are the owner now.

Reply:
Thanks a lot! I'll give this a test.